content/attacks/posts/2018-09-14-Zaif.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2018-09-14
+categories: Custodian
+title: "Zaif loses $60 Million in Cyberattack"
+---
+
+## Summary
+
+In September 2018, Zaif, a Japanese cryptocurrency exchange, [has been hacked](https://cointelegraph.com/news/japanese-cryptocurrency-exchange-hacked-59-million-in-losses-reported). During the hack, the attackers managed to steal about $60 million worth of cryptocurrencies from hot wallets. As a result, Zaif's business was sold to Fisco in order to compensate for the loss of users.
+
+## Attackers
+
+The identity of the attackers has not been publicly disclosed. The investigation into the hack was conducted by the Tech Bureau, the parent company of Zaif, and Japanese law enforcement agencies, but no information has been released regarding the identity of the perpetrators.
+
+## Losses
+
+The total amount of losses from the Zaif hack was approximately $60 million dollars worth of cryptocurrency, including 5.966 bitcoins, as well as Bitcoin Cash and Monacoin.
```

Comment:
Shorten the sentence and make bullet points, and add regarding bitcoin, bitcoin cash and Monacoin worth in USD at the moment, and amounts if possible. Also, it would be better to point token tickers, as BTC instead of 'bitcoins'